plugins/jira-dashboard/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@axis-backstage/plugin-jira-dashboard",
-  "version": "0.7.2",
+  "version": "0.7.3",
```

Comment:
This should not be changed, this will be set when we do a release.

Reply:
Updated